pkgs/top-level/all-packages.nix

```diff
@@ -22714,6 +22714,8 @@ with pkgs;
 
   libedit = callPackage ../development/libraries/libedit { };
 
+  libeduvpn-common = callPackage ../development/libraries/libeduvpn-common { };
```

Comment:
did you try to also put this in pkgs by-name (I think it should also work there)

Reply:
I just tried and you can simply move this `pkgs/development/libraries/libeduvpn-common/default.nix -> pkgs/by-name/li/libeduvpn-common/package.nix` and remove the appropriate line in `pkgs/top-level/all-packages.nix`

Reply:
done! thanks